src/styles/blocks/body.css

```diff
@@ -0,0 +1,5 @@
+body {
```

Comment:
we can not name block after html tag
we can shift this in `normalize.css` or `reset.css`

Reply:
Understand, will remove all the styling from tags to classes, and renamed it.